k8s-scheduler-apiary.tf

```diff
@@ -99,6 +99,9 @@ resource "kubernetes_deployment" "beekeeper_scheduler_apiary" {
             value = base64encode(data.template_file.beekeeper_scheduler_apiary_config.rendered)
           }
         }
+        image_pull_secrets {
```

Comment:
So, this variable is finally used here: https://github.com/ExpediaGroup/apiary-lifecycle/blob/master/files/ecs-container-definition.json#L5

Created here: https://github.com/ExpediaGroup/apiary-lifecycle/blob/master/ecs-templates.tf#L24

So, this is different from how we want to use it in EKS where the value is read from config map. You will need to create a new variable I believe.

Reply:
In https://github.com/ExpediaGroup/apiary-lifecycle/blob/master/ecs-iam.tf#L34, if you set this variable then it will create this IAM policy attachment. Beekeeper is deployed in legacy HCOM environments in ECS, that's why I think it needed this key to read the value from AWS secrets manager.